tests/console/sshd.pm

```diff
@@ -111,9 +112,12 @@ sub check_journal {
         my $journalctl = script_output("journalctl -b -u sshd.service | grep -E 'segfault|fatal'", proceed_on_failure => 1);
         if (is_sle('<15') && $journalctl =~ /diffie-hellman-group1-sha1/) {
             record_info("diffie-hellman-group1-sha1", "Expected message - bsc#1185584 diffie-hellman-group1-sha1 is not enabled on this product");
-        } elsif (is_ppc64le && $journalctl =~ /Timeout before authentication/) {
-            record_info("Timeout before authentication", "bsc#1223178 - [Build 80.1] openQA test fails in sshd: Segfault or fatal journal entry detected in journal");
-        } else {
+        }
+        elsif (is_ppc64le && $journalctl =~ /Timeout before authentication/) {
+            record_info("Timeout before authentication",
+                "bsc#1223178 - [Build 80.1] openQA test fails in sshd: Segfault or fatal journal entry detected in journal");
+        }
+        else {
```

Comment:
There is not point on changing this lines of code, actually the previous version is a shorter style. Additionally you don't need to be ping if this line break in the future.

Reply:
updated, changed here coming from my local perltidy, maybe need update it.